aesm-client/src/lib.rs

```diff
@@ -8,6 +8,10 @@
 //! * `sgxs`. Enable the `sgxs` feature to get an implemention of
 //!   `EinittokenProvider` that uses AESM.
 
+#![doc(html_logo_url = "https://edp.fortanix.com/api/edp-logo.svg",
+       html_favicon_url = "https://edp.fortanix.com/favicon.ico",
+       html_root_url = "https://edp.fortanix.com/api/")]
```

Comment:
Are you sure about the html_root_url? I didn't find any issue with navigation but was hoping to see https://edp.fortanix.com/docs/api/

Reply:
Will fix